modules/i18n/locales.js

```diff
@@ -0,0 +1,772 @@
+    '该标签用户不可见，只有技术支持人员可见。'
+  ],
+  'rank': [
+    'Rank',
+    '排名'
+  ],
+  'remove': [
+    'Remove',
+    '移除'
+  ],
+  'reopen': [
+    'Reopen',
+    '重新打开'
+  ],
+  'reorder': [
+    'Reorder',
+    '调整顺序'
+  ],
+  'resolved': [
+    'Resolved',
+    '已解决'
+  ],
+  'return': [
+    'Return',
+    '返回'
+  ],
+  'role': [
+    'Role',
+    '角色'
+  ],
+  'save': [
+    'Save',
+    '保存'
+  ],
+  'searchKeyword': [
+    'Search keyword',
+    '关键字搜索'
+  ],
+  'selectAll': [
+    'Select all',
+    '全选'
+  ],
+  'settings': [
+    'Settings',
+    '设置'
+  ],
+  'signup': [
+    'Sign up',
+    '注册'
+  ],
+  'staff': [
+    'Staff',
+    '客服'
+  ],
+  'statistics': [
+    'Statistics',
+    '统计'
+  ],
+  'submit': [
+    'Submit',
+    '提交'
+  ],
+  'submittedAt': [
+    'submitted at',
+    '提交于'
+  ],
+  'submitter': [
+    'Submitter',
+    '提交人'
+  ],
+  'submitDate': [
+    'Submit Date',
+    '提交时间'
+  ],
+  'summary': [
+    'Summary',
+    '概要'
+  ],
+  'supportMarkdown': [
+    'Markdown syntax supported',
+    '支持 Markdown 语法'
+  ],
+  'tag': [
+    'Tag',
+    '标签'
+  ],
+  'ticket': [
+    'Ticket',
+    '工单'
+  ],
+  'ticketList': [
+    'Tickets',
+    '工单列表'
+  ],
+  'title': [
+    'Title',
+    '标题'
+  ],
+  'time': [
+    'Time',
+    '时间'
+  ],
+  'type': [
+    'Type',
+    '类型'
+  ],
+  'unresolved': [
+    'Unresolved',
+    '未解决'
+  ],
+  'unconfigured': [
+    'unconfigured',
+    '未设置'
+  ],
+  'until': [
+    'Until',
+    '截止到'
+  ],
+  'updatedAt': [
+    'Updated at',
+    '更新于'
+  ],
+  'user': [
+    'User',
+    '用户'
+  ],
+  'username': [
+    'Username',
+    '用户名'
+  ],
+  // CSStatsUser
+  'notInvoled': [
+    'Not involved',
+    '没有参与'
+  ],
+  'firstReplyTime': [
+    'first reply time',
+    '首次回复时间'
+  ],
+  'averageReplyTime': [
+    'Average reply time',
+    '平均回复时间'
+  ],
+  'replyCount': [
+    'Replies',
+    '回复次数'
+  ],
+  // Categories
+  'newCategory': [
+    'New category',
+    '新增分类'
+  ],
+  // Category
+  'categoryName': [
+    'Name',
+    '分类名称'
+  ],
+  'confirmDisableCategory': [
+    'Confirm disabling this category: ',
+    '确认要停用分类：'
+  ],
+  'parentCategory': [
+    'Parent category (optional)',
+    '父分类(可选)'
+  ],
+  'parentCategoryRequirements': [
+    'You cannot assign this category itself or its subcategories as parent category.',
+    '父分类不能是分类自己或自己的子分类。'
+  ],
+  'ticketTemplate': [
+    'Ticket template',
+    '问题描述模板'
+  ],
+  'ticketTemplateInfo': [
+    'The default content shown when creating a new ticket under this category.',
+    '用户新建该分类工单时，问题描述默认显示这里的内容。'
+  ],
+  // CustomerServiceProfile
+  'associatedAccounts': [
+    'Associated accounts',
+    '账号关联'
+  ],
+  'weCom': [
+    'WeCom',
+    '微信企业号'
+  ],
+  'unlinked': [
+    'Unlinked',
+    '未关联'
+  ],
+  // Vacation
+  'vacation': [
+    'Vacation',
+    '请假'
+  ],
+  'vacationStart': [
+    'Start',
+    '请假开始'
+  ],
+  'backToWork': [
+    'Back',
```

Comment:
```suggestion
    'End',
```
“Start” 对应 ”End“，不易引发歧义。

Reply:
这里对应的中文是「开始工作」，用「End」可能反而有歧义。比如周五请一天假，那 Start 是周五上午， End 是周五下午，Back 是周一上午。 LeanTicket 这里需要填写的是「周五上午」、「周一上午」两个时间点。

Reply:
懂了😅
这样的话可以考虑把英文写完整，比如 “Starting leave” & ”Back to work”。